libaftermath-core/src/defs/aftermath/types/openmp/on_disk.py

```diff
@@ -290,6 +290,26 @@
 
 ################################################################################
 
+am_dsk_openmp_lock_init = EventFrame(
+    name = "am_dsk_openmp_lock_init",
+    entity = "on-disk OpenMP lock init",
+    comment = "Event signalizing init of a lock",
+    fields = FieldList([
+        Field(
+            name = "timestamp",
+            field_type = aftermath.types.builtin.uint64_t,
+            comment = "Time of the event"),
+        Field(
+            name = "wait_id",
+            field_type = aftermath.types.builtin.uint64_t,
+            comment = "ID of the object that is being locked"),
+        Field(
+            name = "kind",
+            field_type = aftermath.types.builtin.uint32_t,
+            comment = "Type of the mutual exclusion event")]))
```

Comment:
Copy-and-paste from mutex events that hasn't been adjusted?

Reply:
Resolved after commits got squashed.